keps/sig-auth/20190607-certificates-api.md

```diff
@@ -132,6 +133,31 @@ A typical successful issuance proceeds as follows.
    the `.Status.Certificate` field.
 1. The requestor observes the update, and stores the certificate locally.
 
+### Signers
+CSRs have a `signerName` field which is used to specify which signer the CSR creator wants to sign the certificate.
+To support migration from v1beta1 to v1, this required field will be defaulted in v1beta1, but not in v1:
+ 1. If it's a kubelet client certificate, it is assigned " kubernetes.io/kubelet-client".
+ 2. Otherwise, it is assign "kubernetes.io/legacy-unknown".
+
+Kubernetes provides the following well-known signers:
+ 1. kubernetes.io/kube-apiserver-client - signs certificates that will be honored as client-certs by the kube-apiserver.
+    Never auto-approved by kube-controller-manager.
+ 2. kubernetes.io/kubelet-client - signs client certificates that will be honored as client-certs by the kube-apiserver.
+    May be auto-approved by kube-controller-manager.
+ 3. kubernetes.io/kubelet-serving - signs serving certificates that are honored as a valid kubelet serving certificate 
+    by the kube-apiserver, but has no other guarantees.  Never auto-approved by kube-controller-manager.
+ 4. kubernetes.io/legacy-unknown - has no guarantees for trust at all.  Some distributions may honor these as client
+    certs, but that behavior is not standard kubernetes behavior.  Never auto-approved by kube-controller-manager.
+
+Distribution of trust happens out of band for these signers.  Any trust outside of those described above are strictly
```

Comment:
I thought we wanted new fields in status to include the CA + intermediates?

Reply:
I want the addition of those fields before v1, but I think that is orthogonal to the spec.signerName field